README.md

```diff
@@ -5,6 +5,9 @@
 ___________________________________________
 
 ## Installation
+
+### Traditional method
+
 1. Add Flathub Beta remote
    ```sh
    flatpak remote-add --user flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
```

Comment:
Maybe remove this suggestion altogether? `RuntimeRepo=` in the `.flatpakref` should just work.

Reply:
Do you know how the .flatpakref behaves when we jump in runtime?  When updating the user will be prompted that he needs to also update the extensions?

Or the suggested method would be to install the .flatpakref everytime we update the runtime?

Reply:
`.flatpakref` will only make sure that non-beta flathub repo is added, thus, avoiding issues like #117
I guess it won't pull in app extensions.

Reply:
Since we are all guessing here, the best I could offer is to put the simple method first. I am heading to #flatpak now